BraintreeDropIn/BTPaymentSelectionViewController.m

```diff
@@ -420,9 +423,10 @@ - (CGFloat)collectionView:(__unused UICollectionView *)savedPaymentMethodsCollec
 - (void)collectionView:(__unused UICollectionView *)savedPaymentMethodsCollectionView didSelectItemAtIndexPath:(__unused NSIndexPath *)indexPath {
     BTUIPaymentMethodCollectionViewCell *cell = (BTUIPaymentMethodCollectionViewCell*)[savedPaymentMethodsCollectionView cellForItemAtIndexPath:indexPath];
     if (self.delegate) {
-        [self.delegate selectionCompletedWithPaymentMethodType:[BTUIKViewUtil paymentOptionTypeForPaymentInfoType:cell.paymentMethodNonce.type] nonce:cell.paymentMethodNonce error:nil];
+        [self.delegate selectionCompletedWithPaymentMethodType:[BTUIKViewUtil paymentOptionTypeForPaymentInfoType:cell.paymentMethodNonce.type]
+                                                         nonce:cell.paymentMethodNonce error:nil];
```

Comment:
To follow this pattern, should error be its own line? I know you didn't do that because it is a small amount of characters, but it might read more consistently.

Reply:
I just missed it. It should definitely go on its own line.